common/src/main/scala/explore/common/ObsQueries.scala

```diff
@@ -133,9 +189,27 @@ object ObsQueries {
   // format: on
   /* END: Generated by clue. Will be replaced when regenerating. */
 
+  private def convertPointing(
```

Comment:
It seems we need to define this for each query that will use our `model` classes, right? Seems a good trade off

Reply:
Yes, that's the idea (the "con" I mentioned in the PR description).